plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -144,18 +144,6 @@ func (g *genericScheduler) Schedule(pod *v1.Pod, nodeLister algorithm.NodeLister
 	return g.selectHost(priorityList)
 }
 
-// Prioritizers returns a slice containing all the scheduler's priority
-// functions and their config. It is exposed for testing only.
-func (g *genericScheduler) Prioritizers() []algorithm.PriorityConfig {
-	return g.prioritizers
-}
-
-// Predicates returns a map containing all the scheduler's predicate
-// functions. It is exposed for testing only.
-func (g *genericScheduler) Predicates() map[string]algorithm.FitPredicate {
```

Comment:
I'm not sure this change will bring clarity, as we are moving out the problem. To me these are related to scheduler algorithm, we should kept here.

Reply:
Kindly ping @anfernee WDYT? Maybe close this one and focus on the other pending PRs?

Reply:
`// It is exposed for testing only.` this especially worries me. Accumulation of things like these are not going to bring value but harm. but anyway, I am closing this.